docs/_toc.yml

```
@@ -61,8 +61,9 @@ parts:
     - file: workshop-8/module-2-presenting-results
     - file: workshop-8/module-3-genomic-surveillance-tanzania
     - file: workshop-8/module-4-genomic-surveillance-ghana
-  - caption: Advanced courses
+  - caption: Advanced Training Materials
     chapters:
+    - file: advanced-training-materials/detecting-population-structure-using-pca
     - file: advanced-training-materials/funestus_exploration
     - file: advanced-training-materials/funestus_pop_struct
     - file: advanced-training-materials/funestus_IR
```

```suggestion
    - file: advanced-training-materials/funestus_exploration
    - file: advanced-training-materials/funestus_pop_struct
    - file: advanced-training-materials/funestus_IR
    - file: advanced-training-materials/detecting-population-structure-using-pca
```

Suggest to add latest at the bottom, to match the W1-8 ordering on the earlier training.